packages/nx/src/command-line/run/run.ts

```diff
@@ -130,7 +131,7 @@ async function printTargetRunHelpInternal(
     targetConfig.options.command
   ) {
     const command = targetConfig.options.command.split(' ')[0];
-    const helpCommand = `${command} --help`;
+    const helpCommand = `${getPackageManagerCommand().exec} ${command} --help`;
```

Comment:
This won't work when the command already has `npx eslint .` in it for some reason?